proposals/0032-type-data.rst

```diff
@@ -2,6 +2,8 @@ Define Kinds Without Promotion
 ==============================
 
 .. proposal-number:: 32
+.. author:: Gabor Greif
```

Comment:
Shouldn’t this be Iavor?

Reply:
Looks like yes. Blame is really messy on that file! I'll fix it. Nice catch!